tests/test_idpi/test_curl.py

```diff
@@ -20,25 +21,26 @@ def print_time(name):
     print(f"{name} took {elapsed:.3f} s")
 
 
-def test_curl(data_dir, grib_defs):
+def test_curl(data_dir):
     datafile = data_dir / "lfff00000000.ch"
     cdatafile = data_dir / "lfff00000000c.ch"
 
-    ds = {}
-    grib_decoder.load_data(ds, ["U", "V", "W"], datafile, chunk_size=None)
-    grib_decoder.load_data(ds, ["HHL", "HSURF"], cdatafile, chunk_size=None)
+    ds = grib_decoder.load_cosmo_data(["U", "V", "W", "HHL"], [datafile, cdatafile])
 
-    dlon = ds["HHL"].attrs["GRIB_iDirectionIncrementInDegrees"]
-    dlat = ds["HHL"].attrs["GRIB_jDirectionIncrementInDegrees"]
-    deg2rad = np.pi / 180
+    geo = ds["HHL"].attrs["geography"]
+    dlon = geo["iDirectionIncrementInDegrees"]
+    dlat = geo["jDirectionIncrementInDegrees"]
+    nj = geo["Nj"]
+    lat_min = geo["latitudeOfFirstGridPointInDegrees"]
 
+    deg2rad = np.pi / 180
+    rlat = get_grid_coords(nj, lat_min, dlat, "y") * deg2rad
     total_diff = TotalDiff(dlon * deg2rad, dlat * deg2rad, ds["HHL"])
-    lat = ds["HHL"]["latitude"] * deg2rad
 
     with print_time("curl"):
```

Comment:
is this intentional or some debugging printouts?

Reply:
It was originally intentional but I can remove it. Now the implementations are more or less equivalent since the introduction of the destagger ufuncs and diff operators.

Reply:
I would remove it or rather have a concept for how to do logging and optionally print timings in all our operators. So that all operators have always the same behaviour, using for example a @operator decorator. But it is not part of this PR, we should create another issue for that